rero_ils/modules/items/mappings/v7/items/item-v0.0.1.json

```diff
@@ -91,6 +91,19 @@
           }
         }
       },
+      "temporary_item_type": {
+        "properties": {
+          "type": {
```

Comment:
Do you need this field `type` here?

Reply:
all '$ref' has this structure into all ES mapping ;-)